main.go

```diff
@@ -129,13 +135,13 @@ func NewNSMClient(ctx context.Context, rootConf *config.Config) networkservice.N
 	// ********************************************************************************
 	// Connect to NSManager
 	// ********************************************************************************
-	var cancel context.CancelFunc
-	ctx, cancel = context.WithTimeout(ctx, 5*time.Second)
+	connectCtx, cancel := context.WithTimeout(ctx, 5*time.Second)
```

Comment:
I know it is not added by this PR, but I guess timeout should be configurable.

Reply:
Yep, added new config parameter.